code/game/objects/items/religion.dm

```diff
@@ -389,3 +389,10 @@
 	force = 15
 	block_power = 30
 	armour_penetration = 5
+
+/obj/item/katana/weak/curator
+	desc = "An ancient Katana. Forged by... Well, it doesn't really say, but surely it's authentic! And sharp to boot!"
+	force = 15
+	block_power = 30
```

Comment:
this should be 0, not even chap gets a katana with this block power. Katanas have weaker blocking, because they are able to block projectiles